src/components/NodesTable/NodesTableItem.vue

```diff
@@ -168,11 +168,8 @@ export default {
 
 .v-theme--light {
   .nodes-table-item {
-    &__version {
```

Comment:
don't merge selectors, keep them separate

Reply:
The selectors may differ in the future, e.g. one selector may require a `margin` or `padding` and other not

Reply:
fair enough